src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,19 @@
+package core.basesyntax;
+
+public class Main {
+    public static void main(String[] args) {
+        FigureSupplier figureSupplier = new FigureSupplier();
+        Figure[] figures = new Figure[6];
+
+        for (int i = 0; i < figures.length / 2; i++) {
+            figures[i] = figureSupplier.getRandomFigure();
+            for (int j = figures.length / 2; j < figures.length; j++) {
+                figures[j] = figureSupplier.getDefaultFigure();
+            }

```

Comment:
not correct. loop inside other loop will have n^2 iterations. fix it (you need to have only 1 loop in this class)